velox/exec/tests/PlanNodeToStringTest.cpp

```diff
@@ -501,3 +502,36 @@ TEST_F(PlanNodeToStringTest, tableScan) {
         plan->toString(true, false));
   }
 }
+
+TEST_F(PlanNodeToStringTest, window) {
+  std::vector<exec::FunctionSignaturePtr> signatures{
+      exec::FunctionSignatureBuilder()
+          .argumentType("BIGINT")
+          .returnType("BIGINT")
+          .build(),
+  };
+  exec::registerWindowFunction("window1", std::move(signatures), nullptr);
+
+  auto plan =
+      PlanBuilder()
+          .tableScan(ROW({"a", "b", "c"}, {VARCHAR(), BIGINT(), BIGINT()}))
+          .window({"window1(c) over (partition by a order by b "
+                   "range between 10 preceding and unbounded following) AS d"})
+          .planNode();
+  ASSERT_EQ(
+      "-- Window[partition by: [a] order by: [b ASC NULLS LAST] "
+      "d := window1(ROW[\"c\"]) RANGE BETWEEN 10 PRECEDING AND UNBOUNDED FOLLOWING] "
+      "-> a:VARCHAR, b:BIGINT, c:BIGINT, d:BIGINT\n",
+      plan->toString(true, false));
+
+  plan = PlanBuilder()
+             .tableScan(ROW({"a", "b", "c"}, {VARCHAR(), BIGINT(), BIGINT()}))
+             .window({"window1(c) over (partition by a "
+                      "range between current row and b following)"})
+             .planNode();
+  ASSERT_EQ(
+      "-- Window[partition by: [a] order by: [] "
```

Comment:
perhaps, drop "order by: []"

Reply:
I think that order by [] makes debugging easier, so feel that we should leave it there. wdyt ?